docs/adminguide/migration/index.rst

```diff
@@ -0,0 +1,113 @@
+
+.. _doc_adminguide_server_migration:
+
+.. cspell:words linenos literalinclude
+
+=======================
+Parsec Server migration
+=======================
+
+This guide covers the migration procedure for the `Parsec`_ server (also called `parsec-backend`).
+
+.. note::
+
+  We consider that you are using the same infrastructure configuration as in the :ref:`Parsec Server installation guide <doc_adminguide_hosting>`.
+  Adapt this guide to your own situation.
+
+.. _Parsec: https://parsec.cloud
+
+Updating to a newer version
+***************************
+
+.. warning::
+
+  Before updating to a newer version, we make sure to back up the database,
+  in case you need to roll back the update.
+
+In this guide, we will migrate ``parsec-backend`` from version ``v2.15.0`` to ``v2.16.0``.
+
+.. _update_docker_image_tag:
+
+1. Update the ``parsec-backend``'s docker image tag of in docker-compose file (``parsec-server.docker.yaml``):
+
+  .. code-block:: yaml
+    :linenos:
+    :emphasize-lines: 5
+
+    services:
+      # ...
+
+      parsec-backend:
+        image: ghcr.io/scille/parsec-cloud/parsec-backend-server:v2.15.0
+        # ...
+
+  You need to change the used tag (at line **5**) to ``v2.16.0``.
+
+2. List the migration that will be applied:
+
+  .. code-block:: bash
+
+    docker compose -f parsec-server.docker.yaml run parsec-backend migrate --dry-run
+
+  The output should look like this:
+
+  .. code-block::
+
+    0001_initial.sql (already applied)
+    0002_add_migration_table.sql (already applied)
+    0003_human_handle.sql (already applied)
+    0004_invite.sql (already applied)
+    0005_redacted_certificates.sql (already applied)
+    0006_outsider_enabled.sql (already applied)
+    0007_users_limit.sql (already applied)
+    0008_apiv1_removal.sql (already applied)
+    0009_add_realm_user_change_table.sql (already applied)
+    0010_add_pki_certificate_table.sql (already applied)
+    0011_add_sequester_tables.sql (already applied)
+    0012_add_sequester_webhook.sql (already applied)
+    0013_add_shamir_recovery.sql (already applied)
+    0014_add_realm_archiving.sql (already applied)
+    0015_some_new_feature.sql (already applied)
+    0016_some_new_feature_2.sql ✔
+
+  .. note::
+
+    This output is provided as an example. Don't expect it to match your output.
+
+  The lines ending with ``already applied`` are migrations already present on the database
+  whereas the one ending with ``✔`` are migrations to be applied.
+
+3. Apply the database migration:
+
+  .. code-block:: bash
+
+    docker compose -f parsec-server.docker.yaml run parsec-backend migrate
+
+.. _restart_parsec_backend_container:
+
+4. Restart the ``parsec-backend`` container:
+
+  .. code-block:: bash
+
+    docker compose -f parsec-server.docker.yaml restart parsec-backend
+
+Rollback to a previous version
+******************************
+
+.. warning::
+
+  **YOU WILL LOSE DATA**
```

Comment:
```suggestion
```

That's already a warning, maybe the `YOU WILL LOSE DATA` statement is a bit too strong

Reply:
Here I clearly want to emphasis that the user will lose data by doing a rollback

Reply:
I still think it's a bit too strong

Reply:
done